hera_librarian/client.py

```diff
@@ -733,3 +733,63 @@ def add_file_row(
                 raise LibrarianError(f"Unknown error. {e}")
 
         return response
+
+    def verify_file_row(
+        self,
+        name: str,
+        size: int,
+        checksum: str,
+        store_name: str,
```

Comment:
As I noted in the main comment thread, we do not necessarily have access to anything other than name; please have the remote librarian check aganist its own database instead of passing a size and checksum.

Store names are entirely transparent to users (they do not and should not care which store their files have been uploaded to), please remove that option. The remote server should check all instances.